scripts/scil_decompose_connectivity.py

```diff
@@ -130,18 +144,18 @@ def build_arg_parser():
     p.add_argument('in_labels',
                    help='Labels file name (nifti). Labels must have 0 as '
                         'background.')
-    p.add_argument('output_dir',
+    p.add_argument('out_hdf5',
                    help='Output directory path.')
```

Comment:
Also, provide extension of the file please. I thought it was .hdf5 but when I see your next script (scil_compute_connectivity),  the input seem to want a .h5 file as input ...